gov.usgs.warc.iridium.sbd.decoder/pom.xml

```diff
@@ -5,7 +5,7 @@
 
 	<groupId>gov.usgs.warc</groupId>
```

Comment:
Can you update:
 * guava to 28.1-jre, 
 * jdt.annotation to 2.2.300
 * spring boot to 2.1.8.RELEASE

Reply:
np